ipn/ipnserver/server.go

```diff
@@ -25,11 +25,20 @@ import (
 	"tailscale.com/logtail/backoff"
 	"tailscale.com/safesocket"
 	"tailscale.com/types/logger"
+	"tailscale.com/version"
 	"tailscale.com/wgengine"
 )
 
+// Options is the configuration of the Tailscale node agent.
 type Options struct {
-	StatePath          string
+	// StatePath is the path to the stored agent state.
+	StatePath string
+	// AutostartStateKey immediately starts the agent using the given
+	// StateKey. If empty, the agent stays idle and waits for a
+	// frontend to start it.
+	AutostartStateKey ipn.StateKey
+	// SurviveDisconnects specifies whether the agent should log out
```

Comment:
That sounds like the opposite of the field name.

When using "whether", the text typically describes the "true" case. If that's not what you meant then say something like "specifies how the agent's connection to foo stays connected when the frontend disconnects. If true, foo. If false, bar."

Reply:
Oops. Fixed.